main/src/assets/lang/fr_FR.json

```diff
@@ -325,48 +325,48 @@
   "Beta": "Beta",
-  "Clone": "Clone",
-  "All the checked Storage will be merged into CasaOS HD.": "All the checked Storage will be merged into CasaOS HD.",
-  "* This feature is now testing and verifying, enabling it may clear personal data and destroy the installed APPs, so it is recommended to use it only during the startup.": "* This feature is now testing and verifying, enabling it may clear personal data and destroy the installed apps, so it is recommended to use it only during the startup.",
-  "Preparing for launch": "Preparing for launch",
-  "to open the app. If it does not work, please restart or try again later.": "to open the app. If it does not work, please restart or try again later.",
-  "This operation may clear all personal data and destroy the installed APPs.": "This operation may clear all personal data and destroy the installed apps.",
+  "Clone": "Cloner",
+  "All the checked Storage will be merged into CasaOS HD.": "L'ensemble des stockage vérifié sera fusionné dans le stockage CasaOS",
+  "* This feature is now testing and verifying, enabling it may clear personal data and destroy the installed APPs, so it is recommended to use it only during the startup.": "* Cette fonction est actuellement testée et vérifiée. Son activation peut effacer des données personnelles et détruire les applications installées, il est donc recommandé de ne l'utiliser qu'au démarrage.",
+  "Preparing for launch": "Préparation au lancement",
+  "to open the app. If it does not work, please restart or try again later.": "pour ouvrir l'application. Si cela ne fonctionne pas, veuillez redémarrer ou réessayer plus tard.",
+  "This operation may clear all personal data and destroy the installed APPs.": "Cette opération peut effacer toutes les données personnelles et détruire les applications installées.",
   "Warning": "Attention",
-  "Click here": "Click here",
-  "Installing app": "Installing app",
-  "Installation completed": "Installation completed",
-  "Starting installation": "Starting installation",
-  "New": "New",
-  "NEW": "NEW",
-  "Check then update": "Check then update",
-  "At least one extra hard drive is needed for this feature.": "At least one extra hard drive is needed for this feature.",
-  "APP may not be available": "App may not be available",
-  "Please": "Please",
-  "{appName} is the latest version!": "{appName} is the latest version!",
-  "{name} has been updated to the latest version!": "{name} has been updated to the latest version!",
-  "Unable to update at the moment!": "Unable to update at the moment!",
-  "Updating": "Updating",
+  "Click here": "Cliquez ici",
+  "Installing app": "Installation de l'application",
+  "Installation completed": "Installation terminée",
+  "Starting installation": "Début de l'installation",
+  "New": "Nouveau",
+  "NEW": "NOUVEAU",
```

Comment:
Duplicate ?
We should not use only one key and make it to uppercase when needed ?

Reply:
you are right.